static/dso_tables.js

```diff
@@ -1,86 +1,186 @@
+        }
+
+        // Link column
+        let cell2_link = row.insertCell(1);
+        for ( let urlName of Object.keys(api.api_urls)) {
+            cell2_link.innerHTML += '<a href="' + api.api_urls[urlName] + '">' + urlName + '</a> ';
+        }
+
+        // Specification column
+        let cell3_Spec = row.insertCell(2);
+        for ( let urlName of Object.keys(api.specificatie_urls)) {
+            cell3_Spec.innerHTML += '<a title="' + urlName + ' specificatie" href="' + api.specificatie_urls[urlName] + '">' + urlName + '</a> ';
+        }
+
+        // Documentation column
+        let cell4_Docs = row.insertCell(3);
+        for ( let urlName of Object.keys(api.documentatie_urls)) {
+            cell4_Docs.innerHTML = '<a title="' + urlName + ' documentatie" href="' + api.documentatie_urls[urlName] + '">' + urlName + '</a> ';
+        }
+
+        // Status column
+        let cell5_Status = row.insertCell(4);
+        cell5_Status.innerHTML = api.beschikbaarheid;
+
+        // License Column
+        let cell6_Licentie = row.insertCell(5);
+        if(api.licentie == "CCBy4.0") {
+            cell6_Licentie.innerHTML =
+                '<a rel="license" href="https://creativecommons.org/licenses/by/4.0/">' +
+                '<img alt="Creative Commons License" src="https://i.creativecommons.org/l/by/4.0/88x31.png"' + 
+                ' width="88" height="31"/></a>';
+        }  else {
+            cell6_Licentie.innerHTML = 
+                '<a rel="license" href="https://creativecommons.org/publicdomain/zero/1.0/">' +
+                '<img src="https://i.creativecommons.org/p/zero/1.0/88x31.png" width="88" height="31" alt="Creative Commons License"/></a>';
+        }      
+    });
+    statusRow.remove();
+ }
+
  
  window.onload = () => {
-     JSONRequest(domain + dsoPath + "?_format=json", tableFromJSON, (resp) => {
-         let statusRow = document.getElementById(tableStatusId);
-         statusRow.firstElementChild.innerText = "Kan datasets niet ophalen.";
-     })
- }
\ No newline at end of file
+    let promises = [
+        JSONRequest("/manual_apis.json").catch(e => {console.log("Kan manual datasets niet ophalen.")}),
+        JSONRequest(domain + dsoPath + "?_format=json").catch(e => {console.log("Kan datasets niet ophalen.")}),
+        JSONRequest(domain + dsoPath + "wfs/").catch(e => {console.log("Kan mvt datasets niet ophalen.")}),
+        JSONRequest(domain + dsoPath + "mvt/").catch(e => {console.log("Kan wfs datasets niet ophalen.")})
+    ]
+    clearSearch();
+    Promise.all(promises).then((results) => {
+        if(results[0]) {
+            parseManualApisJson(results[0]);
+        }
+        if(results[1]) {
+            parseDSOjson(results[1], tables.rest_apis);
+        }
+        if(results[2]) {
+            parseDSOjson(results[2], tables.geo_services, "WFS");
+        }
+        if(results[3]) {
+            parseDSOjson(results[3], tables.tile_services, "MVT");
+        }
```

Comment:
Dit zou ik wat letter doen. misschien `let resultHandler = [  ..., ..., ...]) want als je de promises in een array kan stoppen, passen de results vast ook in een array?